Change steel door texture

An0n3m0us opened the pull request:
Changed the steel door texture to look different compared to the wooden door. Some discussion is going to be required on this, I'm sure.

I modified it ever so slightly to make it have three bars instead of one. The reasoning behind doing this is, firstly, it's a little different from the wooden door shape, and secondly, it looks more like a steel door, and thirdly, it would be nice to have a door in which it's harder to peep through xD

![test3](https://user-images.githubusercontent.com/49789044/74594837-aaa8d680-5032-11ea-9b0e-2aa92cb02d17.png)

paramat commented:
Wow this is nice, big improvement. The wooden door design looks bad, and does not make sense, in steel.

paramat commented:
Just one change needed. Looking at the shading around the 4 windows in the old texture, the dark shading overrides at the corners. This has not been done for the lower windows in the new texture.
Should be:

![doors_door_steel_new_alt](https://user-images.githubusercontent.com/3686677/74595608-89001d00-503b-11ea-9e94-015031d87ede.png)

+1 if done.

An0n3m0us commented:
I changed it because then it would match the top edge. But I don't mind either way; it's a minor difference:

(I've added some effects to the image so one can see the difference easier)
Current <---> Your suggestion
![test1](https://user-images.githubusercontent.com/49789044/74595856-ae425a80-503e-11ea-9796-26c03157c755.png)

paramat commented:
Ok, but your version makes the lower holes look weird and inconsistent. I am -1 for your version because it looks bad.

An0n3m0us commented:
Oh I see what you mean; yeah it kinda does look weird but it's hardly noticeable. I've changed that part back to the original.

paramat commented:
Another possibility could be removing the horizontal bar from your texture to have 4 narrow vertical windows. Some may find your texture overly-armoured, as steel doors are used for many general uses.
But lets see what others think first.

An0n3m0us commented:
> lower holes look weird and inconsistent

Which holes do you mean? I think I might have misunderstood what you were referring to.

An0n3m0us commented:
How about two bars instead of three?

![test](https://user-images.githubusercontent.com/49789044/74633972-d030ff80-515a-11ea-8777-452ec1d08ffb.png)

LoneWolfHT commented:
Heh, that looks a little comical for some reason
Maybe completely remove the window if you do anything

LoneWolfHT commented:
If other people like it I'll pull it into my mtg copy and see if it grows on me

SmallJoker commented:
Maybe the texture needs some more contours and noise? Steel doors can definitely look nice. Here's a modernized version of the John Smith's 32px textures:
![Unbenannt](https://user-images.githubusercontent.com/1497498/74870170-9417bd80-5359-11ea-8ee9-e4b54398de0e.png)

An0n3m0us commented:
Well, I'm terrible at drawing xD

I'll give it a shot though. It definitely needs more noise; it's too plain at the moment.

paramat commented:
We have to be careful here, the texture has to remain somewhat similar in style and colouring, as it is used in so many builds. So SmallJoker's suggestion seems unsuitable as it is a radical change.

sfan5 commented:
I don't have an opinion on the new door texture, but I will say that I disagree with keeping textures similar because they have been "used in many builds".
This argument can be used to prevent any kind of change, even if it is for the better.

If people want their builds to be seen exactly as they built them in 2016, they should keep a copy of MTG from that point. Aiming for unlimited backwards compatibility is not sustainable.

Andrey2470T commented:
The new steel door texture looks like really better than the previous one. I support with three bars.

paramat commented:
sfan5, i mostly agree. My wording was probably not good, i was warning against radical change as suggested by SmallJoker's screenshot, it looks like medieval iron gates with no glass panels.

sorcerykid commented:
I don't think the question is unlimited backwards compatibility, but rather deciding what is a reasonable degree of tolerance for aesthetic changes. In a server environment, for example, the players rarely have _any_ input on the individual textures of nodes, moreless the game version itself.

Add to the fact, quite a few servers are configured to use the Public Remote Media Server by sofar, so mods are automatically sync'd to upstream. That means it's entirely possible that players could login one day and find their cute little homes suddenly look more like prison cells simply because the  steel door was changed without the server administrator's knowledge.

I personally hold a different view: If players dislike the bundled textures for doors, then isn't that all the more incentive for them to seek out a mod from Content DB that offers better variety? For example, my [Extra Doors Mod](https://forum.minetest.net/viewtopic.php?f=9&t=24459) (now integrated into [Doors Redux](https://forum.minetest.net/viewtopic.php?f=9&t=24459)) offers 5 different designs of steel doors. So it's not exactly as if players are locked into one option from Minetest Game.

Anyway, if I had to choose, I would much prefer the version with two vertical bars. I think that is at least a happy compromise and is not so extreme of an alteration :)

rubenwardy commented:
> That means it's entirely possible that players could login one day and find their cute little homes suddenly look more like prison cells simply because the steel door was changed without the server administrator's knowledge.

This is not possible, remote media uses file hashes to identify the images, not image names

paramat commented:
The current texture here is a different style to the wood door, which is good, but i do not think it is suitable for how steel doors are used as it looks too much like a prison door. This needs to be usable for houses or anywhere a lockable door is desired. Needs a different design.
:-1: for current texture.

paramat commented:
Close?
No core dev likes what has been submitted, and no new textures have been submitted foir 4 months.

An0n3m0us commented:
Yeah, I guess so. Core textures are hard to change to something 'better', so I'm not going to bother.

paramat commented:
We can reopen if there is progress.
But generally, i think we should only replace MTG textures if the existing one really needs improving.
